mappings/net/minecraft/GameAndWorldVersion.mapping

```diff
@@ -1,2 +1,4 @@
 CLASS net/minecraft/unmapped/C_xyxorzsk net/minecraft/GameAndWorldVersion
+	METHOD getSeriesId getSeriesName ()Ljava/lang/String;
```

Comment:
Series? What does this mean?

Reply:
![image](https://user-images.githubusercontent.com/72476111/157922456-47641813-6044-404c-a259-ca49282d9e88.png)